console/application.go

```diff
@@ -11,23 +11,20 @@ import (
 )
 
 type Application struct {
-	instance  *cli.App
-	isArtisan bool
+	instance *cli.App
 }
 
-func NewApplication(name, usage, usageText, version string, artisan ...bool) console.Artisan {
+func NewApplication(name, usage, usageText, version string) console.Artisan {
```

Comment:
This was added for `installer`.. NewApplication directly being used in goravel installer.

Reply:
Oh, good catch, No wonder I didn't find the usage in framework.

Reply:
Is it not possible to use env.IsArtisan() for the check? Wouldn’t it be better to avoid relying on explicit parameter passing?

Reply:
Good idea, I think it's possible.

Reply:
Will create another PR to handle this.